packages/help-center/src/components/help-center-header.tsx

```diff
@@ -44,20 +45,32 @@ const SupportModeTitle = () => {
 		}
 	}
 };
+
 const HelpCenterHeader = ( { isMinimized = false, onMinimize, onMaximize, onDismiss }: Header ) => {
 	const unreadCount = useSelect( ( select ) => select( HELP_CENTER_STORE ).getUnreadCount() );
 	const classNames = classnames( 'help-center__container-header' );
 	const { __ } = useI18n();
 	const formattedUnreadCount = unreadCount > 9 ? '9+' : unreadCount;
 
-	const handleClose = () => {
-		onDismiss();
-	};
+	const handleClick = useCallback(
+		( event ) => {
+			if ( isMinimized && event.target === event.currentTarget ) {
+				onMaximize?.();
+			}
+		},
+		[ isMinimized, onMaximize ]
+	);
 
 	return (
 		<CardHeader className={ classNames }>
-			<Flex>
-				<p id="header-text" className="help-center-header__text">
+			<Flex onClick={ handleClick }>
+				<p
+					id="header-text"
+					className="help-center-header__text"
+					onClick={ handleClick }
+					onKeyUp={ handleClick }
```

Comment:
why do we have this key binding? is it needed?

Reply:
I added it because of: https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/bccf0aeab8dd337c5f134f892a6d3588fbc29bdf/docs/rules/click-events-have-key-events.md